<launch>

    <!--Lancer la communication sÃ©rie avec l'arduino-->
    <node name="arduino_serial" pkg="rosserial_python" type="serial_node.py">
    <param name="port" value="/dev/ttyACM0"/>
    </node>

    <!--Lancer le noeud de controle-->
    <node name="base_control" pkg="rufus_master" type="base_control_node.py"/>

</launch>